packages/auth-providers/dbAuth/middleware/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@redwoodjs/auth-dbauth-middleware",
+  "version": "7.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/auth-providers/dbAuth/middleware"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn build:js && yarn build:types",
+    "build:js": "babel src -d dist --extensions \".js,.jsx,.ts,.tsx\" --copy-files --no-copy-ignored",
```

Comment:
Don't think we need to build with babel. We can use the new build system maybe?

Reply:
I can remove but still needs the `.babelrc,js` else tests fail to build when importing types.